src/3rd_party_bundle_remove.c

```diff
@@ -164,26 +190,33 @@ enum swupd_code third_party_bundle_remove_main(int argc, char **argv)
 
 	/* move the bundles provided in the command line into a
 	 * list so it is easier to handle them */
-	for (; *cmdline_bundles; ++cmdline_bundles) {
-		char *bundle = *cmdline_bundles;
-		bundles = list_append_data(bundles, bundle);
+	if (!cmdline_option_orphans) {
+		for (; *cmdline_bundles; ++cmdline_bundles) {
+			char *bundle = *cmdline_bundles;
+			bundles = list_append_data(bundles, bundle);
+		}
+		bundles = list_head(bundles);
 	}
-	bundles = list_head(bundles);
 
 	/* set the command options */
 	bundle_remove_set_option_force(cmdline_option_force);
 	bundle_remove_set_option_recursive(cmdline_option_recursive);
+	bundle_remove_set_option_orphans(cmdline_option_orphans);
```

Comment:
Parameters are better than setting a static variable. It's out of the scope of this PR, but both cmdline_option_force and cmdline_option_recursive could become parameters of execute_remove_bundles_extra